packages/opentelemetry-core/src/context/propagation/B3Propagator.ts

```diff
@@ -26,9 +26,19 @@ import { getParentSpanContext, setExtractedSpanContext } from '../context';
 export const X_B3_TRACE_ID = 'x-b3-traceid';
 export const X_B3_SPAN_ID = 'x-b3-spanid';
 export const X_B3_SAMPLED = 'x-b3-sampled';
+export const X_B3_PARENT_SPAN_ID = 'x-b3-parentspanid';
+export const X_B3_FLAGS = 'x-b3-flags';
+export const PARENT_SPAN_ID_KEY = Context.createKey(
+  'OpenTelemetry Context Key B3 Parent Span Id'
+);
+export const DEBUG_FLAG_KEY = Context.createKey(
+  'OpenTelemetry Context Key B3 Debug Flag'
+);
 const VALID_TRACEID_REGEX = /^([0-9a-f]{16}){1,2}$/i;
 const VALID_SPANID_REGEX = /^[0-9a-f]{16}$/i;
 const INVALID_ID_REGEX = /^0+$/i;
+const VALID_SAMPLED_VALUES = [true, 'true', '1', 1];
```

Comment:
efficiency nit: use a `Set`

Reply:
good point, thanks

Reply:
Thanks. Will add a case for that.

Reply:
I would do whatever brave does in this case.

Reply:
yeah, they ignore casing